packages/local_auth/local_auth/pubspec.yaml

```diff
@@ -5,6 +5,10 @@ repository: https://github.com/flutter/plugins/tree/main/packages/local_auth/loc
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+local_auth%22
 version: 1.1.11
 
+# Temporarily disable publishing to allow moving Android and iOS
+# implementations.
+publish_to: none
+
 environment:
   sdk: ">=2.14.0 <3.0.0"
   flutter: ">=2.5.0"
```

Comment:
You need to change this to 2.8.0 so that the new 2.5 legacy check skips this package (which can't possible resolve since it requires a package that's 2.8.0+).